putils/download-with-progressbar.go

```diff
@@ -43,6 +43,7 @@ func DownloadFileWithProgressbar(progressbar *pterm.ProgressbarPrinter, outputPa
 	counter := &progressbarWriter{}
 	fileSize, err := strconv.Atoi(resp.Header.Get("Content-Length"))
 	if err != nil {
+		out.Close()
```

Comment:
Maybe will be better to make it with defer? In your PR u missed one more place where out was not closet (line 57)

Reply:
I'll add this to the PR, thank you :)